IntelliTectAnalyzer/IntelliTectAnalyzer.Test/AsyncVoidTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TestHelper;
+
+namespace IntelliTectAnalyzer.Tests
+{
+    [TestClass]
+    public class AsyncVoidTests : CodeFixVerifier
+    {
+        [TestMethod]
+        public void AsyncTaskMethod_NoDiagnosticInformationReturned()
+        {
+            var test = @"
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Diagnostics;
+
+    namespace ConsoleApplication1
+    {
+        class TypeName
+        {   
+            public async Task Sample() { }
+        }
+    }";
+
+            VerifyCSharpDiagnostic(test);
+        }
+
+        [TestMethod]
+        public void AsyncVoidMethod_Warning()
+        {
+            var test = @"
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Diagnostics;
+
+    namespace ConsoleApplication1
+    {
+        class TypeName
+        {   
+            public async void Sample() { }
+        }
+    }";
+            var expected = new DiagnosticResult
+            {
+                Id = "INTL0004",
```

Comment:
Looks like the unit test needs to be updated to match.

Reply:
Oh, darn - forgot to run them, my bad...